src/components/grid/index.js

```diff
@@ -1,10 +1,3 @@
-import {
-  Row,
-  Column,
-  Container,
-  FlexColumn,
-  StyledFlipMove,
-  GreyWrapper,
-} from './Grid.styles'
+import { Row, Column, Container, FlexColumn, GreyWrapper } from './Grid.styles'
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Unexpected use of file extension "styles" for "./Grid.styles"](https://app.codacy.com/manual/PrideInLondon/pride-london-web/pullRequest?prid=4658697)